Use OpenGL.framework on macOS if X11 support is not requested.
--- meson.build.orig	2022-02-17 16:56:12
+++ meson.build	2024-11-09 23:48:47
@@ -164,7 +164,7 @@
 
 # Dependencies
 dl_dep = cc.find_library('dl', required: false)
-gl_dep = dependency('gl', required: false)
+gl_dep = dependency('gl', required: false, method: enable_x11 or host_system != 'darwin' ? 'auto' : 'system')
 egl_dep = dependency('egl', required: false)
 elg_headers_dep = egl_dep.partial_dependency(compile_args: true, includes: true)
 
